### PR TITLE
added cli formatting for ls and info

### PR DESCRIPTION
**Pull request by nerdalert**
Formatting cli json and added a few flags for the following subcmds:
network ls
service ls
network info
service info
stripped parentheses from create replies

how-to build/test and results at:
https://gist.github.com/1c49146a6243655a1f06

Signed-off-by: Brent Salisbury <brent.salisbury@docker.com>

### CODE REVIEW

File: client/network.go [hunk 1]
```diff
@@ -129,9 +171,22 @@ func (cli *NetworkCli) CmdNetworkInfo(chain string, args ...string) error {
 	if err != nil {
 		return err
 	}
-	if _, err := io.Copy(cli.out, bytes.NewReader(obj)); err != nil {
+
+	networkResource := &networkResource{}
+	if err := json.NewDecoder(bytes.NewReader(obj)).Decode(networkResource); err != nil {
 		return err
 	}
+	fmt.Fprintf(cli.out, "Network Id: %s\n", networkResource.ID)
```
Comment (mavenugo) [inline client/network.go]: I think we can remove the "Network" word in all the lines. This is output for the `network` command and hence we dont have to print it for every field, We can just remove "Network" from the output

File: client/network.go [hunk 2]
```diff
@@ -129,9 +171,22 @@ func (cli *NetworkCli) CmdNetworkInfo(chain string, args ...string) error {
...
+	fmt.Fprintf(cli.out, "Network Name: %s\n", networkResource.Name)
+	fmt.Fprintf(cli.out, "Network Type: %s\n", networkResource.Type)
+	if networkResource.Endpoints != nil {
+		for _, endpointResource := range networkResource.Endpoints {
+			fmt.Fprintf(cli.out, "Service Id: %s\n", endpointResource.ID)
```
Comment (mavenugo) [inline client/network.go]: Would be good to introduce some intendation using `\t` for services to showcase that the services are part of the network.

File: client/network.go [hunk 3]
```diff
@@ -129,9 +171,22 @@ func (cli *NetworkCli) CmdNetworkInfo(chain string, args ...string) error {
...
+			fmt.Fprintf(cli.out, "Service Name: %s\n", endpointResource.Name)
```
Comment (mavenugo) [inline client/network.go]: Also same comment as before, we dont have to repeat the word "Service" for every field. Rather, we can have a "Service" keyword followed by a bunch of elements such as  "name", "id", etc...

File: client/network.go (hunk 1 above)
Comment (nerdalert) [inline client/network.go]: I left one just so folks can get a clear visual of a Network ID vs. a Service ID.

```
./dnet network service info foo net1
Service Id: 66ba197307d05f8920d376dae43ef9cf78cb38a9cdbe1d9eeba48ed90b819dee
	Name: foo
	Network: net1

./dnet network service info bar net1
Service Id: b790780bacded89bd90330ea36160a9b005c2abfcfb66e81ae2ece5de8a0b58e
	Name: bar
	Network: net1
 
./dnet network info net1
Network Id: 57b3fab23c33f77ac26fdba767fe084257597ef8bfc16bfb375f26cc39359ce5
Name: net1
Type: null
  Service Id: 66ba197307d05f8920d376dae43ef9cf78cb38a9cdbe1d9eeba48ed90b819dee
	Name: foo
	Network: net1
  Service Id: b790780bacded89bd90330ea36160a9b005c2abfcfb66e81ae2ece5de8a0b58e
	Name: bar
	Network: net1
```
Not attached to it by any means but see what you think with it indented now. I did a 2 space and then a \t tab. I liked your suggestion on 2 layers of indentations. It looks quite clear in a terminal.

**Comment by mavenugo** [comment]
Thanks @nerdalert, As we discussed, have pulled in your changes into #206 and resolved all the failures. PTAL

**Pull request closed without merging**